src/inc/quic_platform_winkernel.h

```diff
@@ -989,6 +989,21 @@ NdisSetThreadObjectCompartmentId(
 
 #define CXPLAT_CPUID(FunctionId, eax, ebx, ecx, dx)
 
+//
+// String utility
+//
+
+//
+// Converts a UTF-8 string to a UNICODE_STRING object. NewUnicodeString must be
+// freed with CXPLAT_FREE when done with it.
+//
+QUIC_STATUS
+CxPlatConvertUtf8ToUnicode(
+    _In_z_ const char * Utf8String,
+    _In_ uint32_t Tag,
+    _Out_ PUNICODE_STRING * NewUnicodeString
```

Comment:
Looks like an `_Outptr_` + using `UNICODE_STRING**` would make it clearer.

Reply:
ah, I thought this was the header for the winkernel platform abstraction layer. I see it isn't now. I'll move it ot the correct header